odoo/http.py

```diff
@@ -1164,6 +1164,9 @@ def get_http_params(self):
         params.pop('session_id', None)
         return params
 
+    def get_json_params(self):
+        return json.loads(self.httprequest.get_data(as_text=True))
```

Comment:
Maybe `data` would be better than `params`? or `body`? Not sure about the Best Word (though it's definitely not gusset), but `params` seems a bit iffy there.

Reply:
It is to use the same words as for http `get_http_params`.

Reply:
you're right, let's use `get_json_data`